common/sketch/src/main/java/org/apache/spark/util/sketch/Murmur3_x86_32.java

```diff
@@ -71,6 +73,20 @@ public static int hashUnsafeBytes(Object base, long offset, int lengthInBytes, i
     return fmix(h1, lengthInBytes);
   }
 
+  public static int hashUnsafeBytes2(Object base, long offset, int lengthInBytes, int seed) {
+    // This is compatible with original and another implementations.
+    // Use this method after 2.3.0.
```

Comment:
nit: `Use this method for new components after Spark 2.3`

Reply:
Thanks, fixed it.